biz.aQute.remote/src/aQute/remote/agent/RedirectOutput.java

```diff
@@ -132,4 +135,17 @@ public PrintStream getOut() {
 		return out;
 	}
 
+	public String getLastOutput() {
+		return lastOutput;
+	}
+
+	public void setLastOutput(String out) {
+		if (!"".equals(out) && out != null) {
+			out = out.replaceAll("^>.*$", "");
+
+			if (!"".equals(out) && !out.startsWith("true")) {
```

Comment:
if the shell command returns 'true' , do we not want to just let "true" be returned?

also the "^>.*$" assumes that the gogo prompt will be standard, but what if its modified?  with

```
g! prompt= '$ '
$   
```